pkg/sql/colexec/table_scan/table_scan.go

```diff
@@ -101,12 +101,12 @@ func (tableScan *TableScan) Call(proc *process.Process) (vm.CallResult, error) {
 		// receive topvalue message
 		if tableScan.ctr.msgReceiver != nil {
 			msgs, _ := tableScan.ctr.msgReceiver.ReceiveMessage(false, proc.Ctx)
-			for i := range msgs {
-				msg, ok := msgs[i].(process.TopValueMessage)
-				if !ok {
-					panic("only support top value message in table scan!")
-				}
-				tableScan.Reader.SetFilterZM(msg.TopValueZM)
+			for _ = range msgs {
```

Comment:
Here too.   Why do you comment it out?   Either add enough doc/comment on why, or if the code is dead, just remove.